### PR TITLE
Outputter

### PR DESCRIPTION
**Pull request by Doriaph**
I added the ouputter file and changed a few other things

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -1,11 +1,16 @@
...
+import outputter
 
 ROOT_URL = 'https://www.goodreads.com/quotes/recently_added'
 
 
 def main():
     scraper_agent = scraper.Scraper(ROOT_URL)
-    scraper_agent.scrap()
+    quotes_objects = scraper_agent.scrap()
+    file_path = r'C:\Users\Doria\Documents\ITC\bookworms\books_quotes.csv'
```
Comment (lernerbruno) [inline main.py]: It will only work on your machine :/ Try using relative path

File: outputter.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    def _write_to_csv(self):
+        """Writes the info of the quotes to a csv file."""
+        with open(self.filename, 'a', newline='', encoding='utf-8') as \
+                csv_file:
+            csv_writer = csv.writer(csv_file)
+            if os.stat(self.filename).st_size == 0:  # if the csv file is empty
+                csv_writer.writerow(self.HEADER)
+            for quote in self.quotes_objects:
+                csv_writer.writerow(quote.info)
```
Comment (lernerbruno) [inline outputter.py]: does it append to end of the file?

File: quote.py [hunk 3]
```diff
@@ -25,6 +27,9 @@ def _get_author(self):
         """Extracts info about author from quote html element."""
         text_div = self.html_quote.find('div', class_="quoteText")
         author = text_div.find('span', class_="authorOrTitle").text
+        author = author.strip()
```
Comment (lernerbruno) [inline quote.py]: you can pass a string to strip() with all the chars you want to remove. 
Something like this 
```
author = author.strip(", ")
```

File: scraper.py [hunk 4]
```diff
@@ -1,19 +1,21 @@
 import requests
-from bs4 import BeautifulSoup, NavigableString
+from bs4 import BeautifulSoup
 import quote
```
Comment (lernerbruno) [inline scraper.py]: In order to avoid writing quote.Quote() we can write here : `from quote import Quote`

File: scraper.py [hunk 5]
```diff
@@ -24,19 +26,18 @@ def _get_quotes_elements(self):
...
+        for html_quote in quotes_elements:
+            new_quote = quote.Quote(html_quote)
             quotes_objects.append(new_quote)
         return quotes_objects
 
     def _new_url(self, url, page_num):
         """ Creates a proper URL containing the page number from the input. """
-        page_indicator = '?page='
-        url_new = url + page_indicator + str(page_num)
+        url_new = url + self.PAGE_INDICATOR + str(page_num)
```
Comment (lernerbruno) [inline scraper.py]: I like using string formats, I think its more readable.

File: main.py (hunk 1 above)
Comment (Doriaph) [reply main.py]: fixed it, thanks!

File: quote.py (hunk 3 above)
Comment (Doriaph) [reply quote.py]: you are right, fixed it!

File: scraper.py (hunk 4 above)
Comment (Doriaph) [reply scraper.py]: DONE

File: scraper.py (hunk 5 above)
Comment (Doriaph) [reply scraper.py]: DONE

File: outputter.py (hunk 2 above)
Comment (Doriaph) [reply outputter.py]: yes, this is why I opened the file in an append mode

**Pull request merged**